features/step_definitions/searchAmazon.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'selenium-cucumber'
+require 'selenium-webdriver'
+require 'rspec'
+
+Given 'I search {string}' do |value|
+  wait_for_element_to_display(:id, 'twotabsearchtextbox', 10)
+  wait_for_element_to_display(:xpath, '//input[@value="Go"]', 10)
```

Comment:
Mmmm, I think that wait_for_element_to_enable is correct because element of "Go" is a button (not a visual element).

Reply:
You change the incorrect line :eyes: the button GO should be enable, no the searchbox